### PR TITLE
[13.0][MIG] purchase_request_order_approved

### PR DESCRIPTION
**Pull request by watcharaporn-git**
This module computes the new PO state 'Approved' related to a Purchase
Request Line to display it in the Purchase Request Line tree view and adds a
'Purchase Approved' filter. 
cc @kittiu Can you reviews please.

- [x]  wating purchase_order_approved #829  merge

### CODE REVIEW

File: purchase_request_order_approved/README.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
+===============================
```
Comment (kittiu) [inline purchase_request_order_approved/README.rst:1]: Old version of file.
If you don't want to generate the README file, you can just remove it, and the ocabot will generate for you.

File: purchase_request_order_approved/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+{
+    "name": "Purchase Request Order Approved",
+    "author": "Eficent, " "Odoo Community Association (OCA)",
```
Comment (kittiu) [inline purchase_request_order_approved/__manifest__.py:6]:
```suggestion
    "author": "Eficent,Odoo Community Association (OCA)",
```

File: purchase_request_order_approved/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -0,0 +1,2 @@
+* Gisela Mora <gisela.mora@eficent.com>
+* Lois Rilo <lois.rilo@eficent.com>
```
Comment (kittiu) [inline purchase_request_order_approved/readme/CONTRIBUTORS.rst:2]: You can add yourself as contributor.

**Comment by kittiu** [comment]
Just small comments, otherwise LGTM.

**Review by kittiu — APPROVED** [review]

**Comment by kittiu** [comment]
Ready for review.

**Review by fshah-initos — APPROVED** [review]
Code review and Functional tests. LGTM

**Pull request merged**